src/components/templates/UiQuestion/UiQuestion.vue

```diff
@@ -299,7 +299,7 @@ const defaultProps = computed(() => ({
     ...props.iconInfoAttrs,
   },
   notificationFeedbackAttrs: {
-    type: 'success' as NotificationType,
+    type: 'success' as NotificationProps['type'],
```

Comment:
IMO this cast is unnecessary 🤔 🙂

Reply:
I'll remove this in PR with templates components because then it won't be needed anymore. 👍